atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -128,23 +133,15 @@ public void testLongTableNameLengthFailsNamespace() {
     }
 
     @Test
-    // If you are intentionally making Table API changes, please manually regenerate the ApiTestSchema
-    // and copy the new files to the ${EXPECTED_FILES_FOLDER_PATH} folder.
+    // If you are intentionally making Table API changes, regenerate the ApiTestSchema
+    // with `./gradlew :atlasdb-client:test --tests SchemaTest -Drecreate=true`
```

Comment:
I've improved this test to validate that generated files match exactly what is expected and made it easier to automatically regenerate these files, following the pattern used in our other repos.